src/steps/os/windows.rs

```diff
@@ -212,6 +214,14 @@ pub fn windows_update(ctx: &ExecutionContext) -> Result<()> {
         return powershell.windows_update(ctx);
     }
 
+    let info: os_info::Info = os_info::get();
+    let edition: String = info.edition().unwrap_or("Unknown").to_string();
+    if edition.contains("Windows 11") {
+        print_separator("Windows Update");
+        println!("Windows 11 does not support Windows Update via usoclient.exe. Consider installing PSWindowsUpdate");
```

Comment:
Based on [my previous test](https://github.com/topgrade-rs/topgrade/issues/482#issuecomment-1643109218), this `usoclient` command does not work on Windows 10 either

Reply:
In this case, it is recommended to completely remove usoclient. Previous versions of Windows (7/8), which may still support it, have reached end of life.

Reply:
Makes sense to me